centering content for wider screens

richardayikwei opened the pull request:
I noticed on wider screens the content gets divided into two with a gap between
![Screenshot 2024-11-19 161857](https://github.com/user-attachments/assets/5da07509-9145-4e14-a4e5-e5b01cf67f57)

I added some CSS to center the content on wider screens
![Screenshot 2024-11-19 163458](https://github.com/user-attachments/assets/c32447d1-4065-4965-9f03-b1170b00ee9c)

mattsteen14 commented:
Hey Richard,

Thank you for identifying this. 
This isn't the 1st time this kind of issue has been flagged with one of my sites. I definitely need to factor this into my responsivity checks. I think it's just because I have a regular size MacBook so I don't go beyond when it's full size.

I think you just need to look at the max width 800px media query because when the aside is hidden the post card is overflowing a bit.

Current:
<img width="506" alt="Screenshot 2024-11-20 at 10 56 43" src="https://github.com/user-attachments/assets/ac317bf9-ad92-4bf0-9050-bfa6db21e828">

Your commit:
<img width="509" alt="Screenshot 2024-11-20 at 10 58 00" src="https://github.com/user-attachments/assets/89054688-2934-4c7d-855c-c5c99f17b682">

If you can fix this to make it look like image 1 on smaller screens then I would be happy to approve and merge.

richardayikwei commented:
![Screenshot 2024-11-20 125610](https://github.com/user-attachments/assets/29d9bde7-c7b8-4140-a860-71fedc2d5396)
Thanks for prompt review. I have added a media query to handle 1280px and above screens sizes. I have removed the initial CSS that was added to the app class that was causing the problem.

mattsteen14 reviewed (approved):
Thank you so much. Adding a media query for bigger screens works great & you solved the issue with smaller screens. Happy to approve & merge.
